infrastructure/variables.tf

```diff
@@ -1,4 +1,8 @@
-variable "product" {}
+variable "product" {
+  type        = string
+  description = "The name of your application"
+  default     = "civil-sdt"
+}
 
 variable "component" {}
```

Comment:
The type of this variable is `string`.  Also, probably a good idea to add the type to other variables where it's not specified so that they are defined in a consistent way.

Reply:
added type to some of the fields

Reply:
Confirmed that type added to fields.